data/map planets.txt

```diff
@@ -3061,6 +3080,12 @@ planet "New Kansas"
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "New Kola"
+	attributes uninhabited
+	landscape land/haze2
+	description `At a distance, New Kola appears to be a gas giant; yet upon approach you notice that the thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only a breathing apparatus, and even then the crushing gravity would cause serious problems if you stayed for long.`
```

Comment:
```suggestion
	description `At a distance, New Kola appears to be a gas giant; yet upon approach you notice that the thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only a breathing apparatus, and even then the pressure and crushing gravity would cause serious problems if you stayed for long.`
```

There are three things here.

1. Gravity is mentioned to be “crushing.” Super earths are generally about 2-10x the mass and around the same radius as the Earth, so I can assume about 2-10x the gravity. I will assume it is about 5x as much, which is beyond the range of what humans can tolerate.

2. Surface pressure is greater than Earth. However, there are **many** factors that control surface pressure, with gravity only being one of them. Atmospheric composition, atmospheric thickness, and altitude are the main ones, however, temperature also controls pressure due to the ideal gas law. As temperature increases, pressure decreases. Given Mars gravity and atmospheric composition, a pressure of 35 atm on the surface and a surface temperature of 95°C leads to about 22.0 atm, or 15.5 atm for Earth and 13.2 atm for Venus ([source](https://www.mide.com/interplanetary-air-pressure-at-altitude-calculator)). Higher gravity leads to a slower decrease in pressure, whereas a more dense atmosphere leads to a faster decrease in pressure, so Mars has a slower decrease in pressure as its atmosphere is so thin. As a super Earth would have higher gravity and a more dense atmosphere, you could argue that the density somewhat counters the higher gravity. The higher gravity itself would lead to a decrease of 1/5 as much with Earth atmospheric density, leading to about 31 atm. The difference in pressure change between Venus and Earth means that with Venus composition, pressure would be 26 atm. So, pressure would probably be something like 28 atm, which the same as the experienced at around 280m underwater. With breathing equipment, it *is* possible to go 280m underwater, so I don’t think pressure like this is too bad for the constraints given. 

4. For temperature, depending on the atmospheric composition, it can actually increase with elevation. On Earth, this happens in the stratosphere, due to the presence of ozone. The implication is that there is no oxygen and therefore no ozone layer, meaning no temperature inversion causing the temperatures to increase at some elevation, so the temperatures will just continuously decrease until the atmosphere becomes so thin that temperatures increase. Using the dry adiabatic lapse rate, there would be pleasant temperatures at around 8 km up. However, that’s assuming 90°C temperatures. The planet appears to be in the habitable zone, so some extra greenhouse gases leading to warmer temperatures isn’t that hard to come up with.

Basically, the pressure would be high, but the criteria is tolerable considering you are using specialized breathing equipment (“oxygen gear”), and while 35 atm is not tolerable, 28 atm can be, as the limit is around 30 atm. Of course, you still probably wouldn’t want to stay very long, but you can step outside. I would probably modify the final sentence slightly, but I don’t think that is necessary.

Reply:
> Gravity is mentioned to be “crushing.” Super earths are generally about 2-10x the mass and around the same radius as the Earth, so I can assume about 2-10x the gravity. I will assume it is about 5x as much, which is beyond the range of what humans can tolerate.

I thought large terrestrial planets had similar densities to earth, but larger masses and radii, and higher surface gravity - after all, rock isn't compressible like gas is, so only changing the composition would change the density. Of course, mass is proportional to the cube of radius (for a given density), so with 2x the radius you'd have 8x the mass, and hence 4x the surface gravity. That's probably just a little bigger than New Kola is, and it might be made of rock with a slightly lower average density than Earth, so maybe surface gravity is something like 3.5 times Earth gravity.

> So, pressure would probably be something like 28 atm, which the same as the experienced at around 280m underwater. With breathing equipment, it is possible to go 280m underwater, so I don’t think pressure like this is too bad for the constraints given.

I didn't realise humans could survive anything like that much atmospheric pressure - I'd have guessed the upper limit was more like 10atm. Still, I think you're right in saying that the higher gravity would make atmospheric pressure drop off more quickly with altitude, so getting from 30atm at ground level down to 1-2 atm at the peaks of the highest mountains doesn't seem too absurd? Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

Reply:
> . Still, I think you're right in saying that the higher gravity would make atmospheric pressure drop off more quickly with altitude

Less quickly, as the surface pressure would be more from gravity and less from a dense atmosphere. Compare Earth to Jupiter with all values being the same. For 3x as much gravity, pressure drops at 1/3 the rate.

> I didn't realise humans could survive anything like that much atmospheric pressure - I'd have guessed the upper limit was more like 10atm

Divers can go more than 100m underwater, though very few have gone more than 300m underwater, so 30 atm is basically the cutoff. The main issue is not inhaling the wrong oxygen pressure, but your oxygen gear would likely have the right concentrations to counter this.

> Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

30 atm would lead to 17.7-21.1 atm for Earth/Venus with 3x gravity, a 20000m altitude, and surface temperatures of 90°C, so around 18-19 atm. That isn’t close to 1 atm due to the stronger gravity, but it is close to 15 atm, which is how much the ship can definitely withstand, so 20km seems reasonable to me with close to 30 atm pressure.

Reply:
> Of course, mass is proportional to the cube of radius (for a given density), so with 2x the radius you'd have 8x the mass, and hence 4x the surface gravity.

Because you're twice as far away from the centre, the surface gravity will only be twice.

Earth surface gravitational force = G \* M_e / (R_e)^2
Where:
- G is the gravitational constant
- M_e is Earth's mass.
- R_e is Earth's radius.

If we consider a planet with the same density as Earth with twice the radius, it's mass will indeed be 8 times Earth's mass.
So, gravitational force on this planet's surface = G \* 8 \* M_e / (2 \* R_e)^2 = (8 / 4) \* G \* M_e / (R_e)^2 = 2 * G \* M_e / (R_e)^2

I'm not doing anymore maths right now, but I do intend to apply some realism to the discussions here at some point (unless someone else does it first.)

Reply:
> Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

Keep in mind that Olympus Mons is taller than mountains on Earth in part because Mars has lower gravity. Lower gravity allows mountains to grow taller, as it takes more height for them to collapse under their own mass. A planet with a higher gravity will be likely have smaller mountains.

> As a super Earth would have higher gravity and a more dense atmosphere

Need a super earth necessarily have a more dense atmosphere?

Reply:
A Super Earth only means it has a mass of 2-10x that of Earth, and can also mean a radius of around 0.8-1.2x that of Earth.